model/riscv_insts_mext.sail

```diff
@@ -173,13 +173,13 @@ mapping clause assembly = REM(rs2, rs1, rd, s)
 /* ****************************************************************** */
 union clause ast = MULW : (regidx, regidx, regidx)
 
-mapping clause encdec = MULW(rs2, rs1, rd)
+mapping clause encdec = MULW(rs2, rs1, rd) if extension("M")
       if sizeof(xlen) == 64
   <-> 0b0000001 @ rs2 @ rs1 @ 0b000 @ rd @ 0b0111011
       if sizeof(xlen) == 64
```

Comment:
This doesn't compile. You need to combine the new condition with the existing condition, and on both sides of the mapping, like this:
```
mapping clause encdec = MULW(rs2, rs1, rd)
      if extension("M") & sizeof(xlen) == 64
  <-> 0b0000001 @ rs2 @ rs1 @ 0b000 @ rd @ 0b0111011
      if extension("M") & sizeof(xlen) == 64
```